test/api_spec.coffee

```diff
@@ -611,6 +617,20 @@ describe "api", ->
         true
       true
 
+    it "should successfully call update api with 'quality_override'", (done) ->
+      @timeout helper.TIMEOUT_LONG
+      options = {
+        quality_override: 'auto:best',
+        moderation_status: "approved"
```

Comment:
Why is this test mixed with moderation?

Reply:
We just based it on other examples we've seen here and there, to test with other functionality, many of your tests are written like this.